client/me/purchases/membership-item/index.tsx

```diff
@@ -9,86 +9,98 @@ import formatCurrency from '@automattic/format-currency';
  * Internal dependencies
  */
 import { CompactCard } from '@automattic/components';
-import Gridicon from 'calypso/components/gridicon';
 import { useLocalizedMoment } from 'calypso/components/localized-moment';
 import { MembershipSubscription } from 'calypso/lib/purchases/types';
+import SiteIcon from 'calypso/blocks/site-icon';
 
 /**
  * Style dependencies
  */
-import './style.scss';
+import 'calypso/me/purchases/style.scss';
 
 const MembershipTerms = ( { subscription }: { subscription: MembershipSubscription } ) => {
 	const translate = useTranslate();
 	const moment = useLocalizedMoment();
 
-	/* $5 - never expires. */
 	if ( subscription.end_date === null ) {
-		return (
-			<div className="membership-item__term-label">
-				{ translate( '%(amount)s - never expires.', {
-					args: {
-						amount: formatCurrency( Number( subscription.renewal_price ), subscription.currency ),
-					},
-				} ) }
-			</div>
-		);
+		return translate( 'Never expires' );
 	}
 
-	/* Renews every month for $5. Next renewal on November 22, 2020. */
-	if ( subscription.renew_interval ) {
-		return (
-			<div className="membership-item__term-label">
-				{ translate( 'Renews every %(interval)s for %(amount)s. Next renewal on %(date)s.', {
-					args: {
-						interval: subscription.renew_interval,
-						amount: formatCurrency( Number( subscription.renewal_price ), subscription.currency ),
-						date: moment( subscription.end_date ).format( 'LL' ),
-					},
-				} ) }
-			</div>
-		);
-	}
+	return translate( 'Renews at %(amount)s on %(date)s', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Renews at %(amount)s on %(date)s.' )` *ES Score: 7*

Reply:
Same term but no punctuation at the end.